src/pages/signin/SignInPage.js

```diff
@@ -77,9 +80,10 @@ function getRenderOptions({hasLogin, hasValidateCode, hasAccount, isPrimaryLogin
     };
 }
 
-function SignInPage({credentials, account}) {
+function SignInPage({credentials, account, isInModal}) {
     const {translate, formatPhoneNumber} = useLocalize();
     const {isSmallScreenWidth} = useWindowDimensions();
+    const showSmallScreen = isSmallScreenWidth || isInModal;
```

Comment:
NAB: We should use the `should` or `is` prefix for the boolean value.

Reply:
updated to shouldShowSmallScreen